src/libraries/Microsoft.PowerFx.Core/Entities/IRefreshable.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Entities
+{
+    public interface IRefreshable
```

Comment:
this needs some comment - at least to note who calls it?  Ie, who should implement it?

Reply:
Always a good idea to have comments